viewer/core/src/public/migration/types.ts

```diff
@@ -191,6 +190,29 @@ export interface AddModelOptions {
   geometryFilter?: GeometryFilter;
 }
 
+export type CameraControlsOptions = {
+  /**
+   * Sets mouse wheel initiated action based on either lerping in the direction of the cursor or using so-called scroll target notion.
+   * When mouse wheel scroll is initiated scroll target is set to the point on the model where cursor is hovering over. Then when zooming is happening camera
+   * target moves towards scroll target with speed proportional to zooming speed. This prevents the camera to go through the model when using mouse navigation,
+   * but keyboard navigation still allows to go through the model.
+   *
+   * Default is 'zoomPastCursor'.
+   *
+   */
+  mouseWheelAction?:
+    | 'zoomToTarget' // mouse wheel scroll zooms just to the current target (center of the screen) of the camera
+    | 'zoomPastCursor' // mouse wheel scroll zooms in the direction of the ray coming from camera through cursor screen position, allows going through objects
+    | 'zoomToCursor'; // mouse wheel scroll zooms towards the point on the model which cursor is hovering over, doesn't allow going through objects
```

Comment:
The descriptions you have inline here are very useful and should be placed in the `/** ... */`-doc above so it's also accessible to the users.